ArticleTemplates/assets/scss/type/_media-gallery.scss

```diff
@@ -219,6 +223,15 @@
 }
 
 .is_advertising.garnett--type-guardianlabs[data-content-type='gallery'] .article {
+    p a, em a {
+        color: #5fa794;
```

Comment:
Do we have a SASS variable for this colour?

Reply:
We did not have a SASS variable, i will add one 👍